antismash/modules/terpene/html_output.py

```
@@ -0,0 +1,99 @@
+from collections import OrderedDict
+from typing import Any
+
+from antismash.common import path
+from antismash.common.html_renderer import FileTemplate, HTMLSections, Markup, docs_link
+from antismash.common.layers import OptionsLayer, RegionLayer, RecordLayer
+
+from .results import DomainPrediction, ProtoclusterPrediction, TerpeneResults
+from .terpene_analysis import _load_hmm_properties
+
+_hmm_properties = _load_hmm_properties()
+
+def will_handle(_products: list[str], categories: set[str]) -> bool:
+    """ Returns true if one or more relevant products or product categories are present """
+    return "terpene" in categories
+
+def get_domain_description(prediction: DomainPrediction) -> str:
+    description = _hmm_properties[prediction.type].description
+    return description
+
+def format_subtype(prediction: DomainPrediction) -> str:
+    if not prediction.subtypes:
+        if _hmm_properties[prediction.type].subtypes:
+            return "unknown"
+        return "none"
+    descriptions = []
+    for subtype in prediction.subtypes:
+        try:
+            short_description = "".join(_hmm_properties[subtype].description.split("; ")[1:])
+        except:
```

Bare `except` is dangerous, since it can also swallow syntax errors.